lrpar/src/lib/ctbuilder.rs

```diff
@@ -920,7 +927,7 @@ where
             if !grm.rule_to_prods(ridx).contains(&grm.start_prod()) {
                 write!(
                     outs,
-                    "    #[allow(dead_code)]\n    pub const R_{}: {} = {:?};\n",
+                    "    #[allow(dead_code)]\n    pub const R_{}: {} = {};\n",
```

Comment:
two things to note here,

1. There are other places we are encoding usize with plain old `{}` 
2. quote does the wrong thing here, as it will encode type specifies e.g. `0usize` or whatever
and we want this to be something of `type_name::<StorageT>()` below.

As such, I believe it is okay to just remove this debug format

Reply:
I went ahead and made a newtype wrapper in [484abd1](https://github.com/softdevteam/grmtools/pull/494/commits/484abd1f6a4196453beccfac96a875c4f9286ce0) that does this unsuffixed printing, or at least shows how it would be achieved with `ToTokens` instead of format printing If we wanted to go that route.

Reply:
Works for me!